ClientApp/src/components/shared/NavBar/NavMenuAdmin.tsx

```diff
@@ -28,6 +28,7 @@ export default class NavMenuAdmin extends React.PureComponent<
           <Container>
             <NavbarBrand tag={Link} to="/">
               <img className={"logo-small"} src={logo}></img>
+              Hei Institution
```

Comment:
Is it supposed to say "Hei Institution" ? 
This will render if role is Institution or Admin.

Reply:
This was for testing and forgot to delete it, nice catch 👍